packages/backend/package.json

```diff
@@ -185,6 +185,7 @@
   "dependencies": {
     "@sap-devx/webview-rpc": "0.3.1",
     "@sap-devx/yeoman-ui-types": "^1.12.2",
+    "@sap-ux/environment-check": "0.15.64",
```

Comment:
Please do not add dependencies on @sap-ux we want the OSS to remain clean of such dependencies

Reply:
Ok, I've removed this and add the code to the npm util.